### DISCUSSION TITLE
It works on Linux!

### DISCUSSION THREAD

**Discussion by HengYeDev** [perpetualKid/FreeTrainSimulator#53] [Show and tell]
![image](https://user-images.githubusercontent.com/62600441/112333481-0790aa80-8c91-11eb-9ad6-5f89aa00e307.png)

**Comment by HengYeDev** [comment]
Limitation: "Logging" must be OFF to run. Performance seems faster than regular version.

**Reply by perpetualKid** [reply]
this is great to hear, really awesome! At least when comparing on Windows, performance of this version is fairly improved in many places, but there seems some impact whether using Integrated or Dedicated Graphics. Do you have any figures to compare, FPS?

Is this the .NET Core version or the .NET Framework version you got to run now? I was looking into the hardware info stuff today which is breaking/depending on WMI so far, if that is the only/major issue for Linux we could easily replace to get logging operational again.

**Reply by HengYeDev** [reply]
This is .NET Core version. I have 130 FPS on intel integrated graphics and 160 FPS on nvidia gtx 1650 TI. It was about 30-50 FPS in the regular version of Open Rails. Buttery smooth, thanks for making this

**Comment by HengYeDev** [comment]
It seems that the screenshot feature is a bit buggy. This is a picture taken with the native screenshot feature. The above one is from open rails built in.
![image](https://user-images.githubusercontent.com/62600441/112334729-057b1b80-8c92-11eb-9f27-aeb839610235.png)

**Comment by HengYeDev** [comment]
![image](https://user-images.githubusercontent.com/62600441/112663998-d69aac00-8e2f-11eb-8655-fb3ae886c65c.png)

**Reply by perpetualKid** [reply]
mind writing a short summary about the prerequisites and settings and share here, or add in the wiki?

**Reply by HengYeDev** [reply]
> mind writing a short summary about the prerequisites and settings and share here, or add in the wiki?

I've got it in the wiki already :) @perpetualKid

**Comment by HengYeDev** [comment]
https://github.com/perpetualKid/ORTS-MG/wiki/Linux-Wine

**Reply by perpetualKid** [reply]
sorry didn't see that earlier, there is no notification for wiki updates. Appreciate the effort.
